apps/frontend/src/app/1_atoms/ErrorRenderer/ErrorRenderer.tsx

```diff
@@ -20,6 +26,12 @@ export const ErrorRenderer: FC<ErrorRendererProps> = ({
 }) => (
   <div className="container flex flex-col justify-center items-center flex-grow py-12">
     <div className="text-center whitespace-normal px-4">
+      <div className="w-full flex items-center justify-center">
+        <div className="w-64">
+          <Lottie animation="loading" />
```

Comment:
```suggestion
          <Lottie animation="error" />
```
I think you should use the new `error` animation here instead of `loading`. Also, you don't use the new `404` animation anywhere.

Reply:
`404` should be used on line 25 here:
`apps\frontend\src\app\5_pages\NotFoundPage\NotFoundPage.tsx`